test/amqp.test.js

```diff
@@ -0,0 +1,41 @@
+const chai = require('chai');
```

Comment:
I don't see any value in this test. 
For me, it would be better to test this in the happy path which will assert that the correct AQMP has been sent for the specific flow and the message is in the correct format.

Reply:
The test checks if amqp.js sendMessage is sending messages correctly in the queue. I will add it to the happy path to be ta part of the whole flow.